modules/openapi-generator/src/main/resources/csharp-netcore/ClientUtils.mustache

```diff
@@ -1,6 +1,7 @@
 {{>partial_header}}
 
 using System;
+using System.Globalization;
```

Comment:
Can we add this line after `using System.Collections;` to follow the alphabetical order?

Reply:
Sure :) https://github.com/OpenAPITools/openapi-generator/pull/6194/commits/39dffb83590addc09fd63d04d0393e230ad97ba3